### DISCUSSION TITLE
Question on GFDL Vortex Tracker installation

### DISCUSSION THREAD

**Discussion by AustinConaty** [dtcenter/METplus#1781] [Installation] [answered]
Greetings,
I went to this web page:
https://dtcenter.org/community-code/gfdl-vortex-tracker/download
and downloaded the 
standalone_gfdl-vortextracker_v3.9a.tar.gz
and
test_data.tar.gz

I followed the instructions in the Users Guide here:
https://dtcenter.org/sites/default/files/community-code/gfdl/standalone_tracker_UG_v3.9a.pdf

I installed and then pointed to the Png Zlib and Jasper libraries.
I got 4 out of the 5 expected executables when I compiled but failed
to get the tracker executable itself, gettrk.exe
grb2index.exe  grbindex.exe  tave.exe  vint.exe all compiled okay.

I don't see a lot of GFDL tracker traffic here, but anyone have a similar experience?

Thanks,
Austin Conaty

**Comment by georgemccabe** [comment, marked as answer]
Hi @AustinConaty, there have been a few other discussions regarding the GFDL Tracker, like #1624. The tracker is an external tool that the METplus team did not develop, so it is difficult for us to provide support. I see that the link in GFDL Tracker download page for support is broken, but I do see that the PDF mentions to send any questions to hwrf-help@ucar.edu. I would recommend sending an email there. I'm sorry that we aren't able to provide more help.

**Reply by AustinConaty** [reply]
Thank you George.  I was hoping someone had stumbled with the installation like I am.  I'll get it figured out.
Looking forward to running the vortex tracker so we can use the TC tools that we learned about.
Thanks for all you do!

**Reply by georgemccabe** [reply]
If you are able to use Docker, there is a [Docker image available on DockerHub](https://hub.docker.com/layers/dtcenter/metplus-envs/gfdl-tracker/images/sha256-86ff1c72012c2488bd68e97da9177d9a15aa69a6ad54828929925564b72dfbfc?context=repo) that I have created that has the GFDL Tracker installed. You can mount your input data to the Docker container and mount an output directory so the files can be made available on your machine.

`docker run --rm -it dtcenter/metplus-envs:gfdl-tracker`

The tracker executables can be found in /standalone_gfdl-vortextracker_v3.9a_gcc/trk_exec
